scripts/docker/helper.sh

```diff
@@ -43,6 +43,18 @@ function export_suffixes () {
             ;;
         esac
         ;;
+        lightnet)
```

Comment:
This seems orthogonal to the PR; can you split it out?

Reply:
Agree, Dariusz just created a PR for that 🎉 

https://github.com/MinaProtocol/mina/pull/16084